11ty/definitions/mansplaining.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Mansplaining
+slug: mansplaining
+speech: verb
+defined: true
+reading:
+- text: "Rebecca Solnit's 'Men Explain Things to Me' (2008)"
+  href: https://www.guernicamag.com/rebecca-solnit-men-explain-things-to-me/
+- text: "Kim Goodwin's 'Am I mansplaining? graphic"
+  href: https://ichef.bbci.co.uk/wwfeatures/live/1600_900/images/live/p0/6f/zj/p06fzjxg.jpg
+---
+
+A man explaining a topic to a woman in a condescending, overconfident way, without acknowledging the fact that the woman often knows more about that topic than the man doing the explaining. 
+
+Examples include but aren't limited to: a man explaining to a woman how to pronounce her own name, explaining menstruation, or explaining the history of a country with which the woman is associated but the man has never been to.  
+
+## Note
+
+A good way to check if you're mansplaining is to ask yourself whether: 
+
+(1) the person wants the explanation
```

Comment:
```suggestion
(1) the person explicitly asked for the explanation
```

Reply:
I think this shift of language emphasises consent-based explanations; I would venture to say a lot of mansplainers would incorrectly assume their subjects would want the explanation 😂😭